src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -927,27 +999,42 @@ private synchronized void update()
             logger.debug("Prioritizing endpoints, adjusted last-n: " + adjustedLastN +
                 ", sorted endpoint list: " +
                 conferenceEndpoints.stream().map(AbstractEndpoint::getID).collect(Collectors.joining(", ")) +
-                ". Selected endpoints: " + String.join(", ", selectedEndpointIds) +
-                ". Pinned endpoints: " + String.join(", ", pinnedEndpointIds));
+                ". Endpoints constraints: " + Arrays.toString(videoConstraintsMap.values().toArray()));
         }
 
-        int endpointPriority = 0;
-
-        // First, bubble-up the selected endpoints (whoever's on-stage needs to
-        // be visible).
-        for (Iterator<AbstractEndpoint> it = conferenceEndpoints.iterator();
-             it.hasNext() && endpointPriority < adjustedLastN;)
+        // Before endpoint constraints we had had the notion of selected and
+        // pinned endpoints. "selected" endpoints were endpoints that had
+        // 720p ideal height and prioritized over everything else and were
+        // allocated bandwidth first (up to their preferred resolution).
+        // "Pinned" endpoints where those thee had 180p ideal height and were
+        // ranked lower than selected endpoints but before the rest of the pack.
+        // Finally, any remaining endpoints were ranked after pinned endpoints.
```

Comment:
I don't think we need a comment describing what the code used to do -- that's what git history is for.  Better to describe what it's doing now.